samples/net/sockets/echo_async/src/socket_echo.c

```diff
@@ -143,14 +149,18 @@ int main(void)
 	if (res == -1) {
 		printf("Cannot bind IPv6, errno: %d\n", errno);
 	}
+	num_servs++;
+#endif
 
 	setblocking(serv4, false);
-	setblocking(serv6, false);
 	listen(serv4, 5);
-	listen(serv6, 5);
-
 	pollfds_add(serv4);
+
+#ifdef CONFIG_NET_IPV6
```

Comment:
Would be nice to minimize number of #ifdef's, and arrange complete setup of ipv4 socket, then complete setup of ipv6 socket (wrapped in #ifdef).

Reply:
Done